pkgs/applications/networking/browsers/google-chrome/default.nix

```diff
@@ -39,6 +39,9 @@
 
 , gsettings-desktop-schemas
 , gnome3
+
+# For video acceleration under libva
+, libvaSupport ? true, libva ? null
```

Comment:
```suggestion
, libvaSupport ? true, libva
```
the package should not be null.

Reply:
Ok will update. I was following the pattern that I've above and elsewhere.

Reply:
So... When should someone use `? null` vs not?

Reply:
Might depend on the reviewer but AFAIK most are currently trying to avoid it whenever possible (i.e. almost always).